docs/UserGuide.md

```diff
@@ -3,436 +3,713 @@ layout: page
-* On macOS, press **Cmd + Space**, type **`Terminal`** into the search bar, and press **Enter**.
-* On Linux, open your Terminal application.
+# 1. Introduction
+## 1.1 What is AgentAssist?
 
-### 5. Run the applicaton
+AgentAssist is the **definitive desktop tool for credit card sales agents**. Merging the swift efficiency of a Command Line Interface (CLI) with the intuitive accessibility of a Graphical User Interface (GUI), this application lets you manage contact databases, track sales progress, and execute transactions with unprecedented speed. 
 
-Navigate your terminal to the folder where you saved the AgentAssist application:
+**Overview of Key Features:**
+* **Contact Management**: 
+  * Manage your client details easily. Add, edit, and delete contacts to keep all your client information in one accessible place.
+* **Keyboard-only Navigation**: 
+  * Navigate through the application entirely via keyboard shortcuts, improving workflow efficiency.
+* **Multi-Level Filtering**: 
+  * Filter your data by multiple criteria to find exactly who you’re looking for.
+* **Auto-Save**: 
+  * Automatically saves your work as you go, ensuring data is updated without manual intervention.
+* **Effortless Data Import & Export**: 
+  * Import or export client and sales data in compatible formats for backups or use in other systems.
 
-* Type **`cd `** (with a space after it), then drag and drop the folder where you placed the `.jar` file into the terminal window. The command should look similar to this: `cd '/Users/name/AgentAssistFolder'`. Press Enter.<br>
+Maximize your productivity, minimize your response time, and amplify your sales performance. With AgentAssist, you're not just keeping up with the competitive world of credit card sales — _you're setting the pace_.
 
-Type the following command: **`java -jar agentassist.jar`** and press **Enter**.
+--------------------------------------------------------------------------------------------------------------------
 
-* A window similar to the below image should appear in a few seconds. You will see a graphical user interface with sample contact information already added.<br>
-  ![Ui](images/Ui.png)
+# 2. Important Prerequisites
 
----
+Before you start using AgentAssist, there are a few prerequisites to ensure you get the most out of the application:
 
-## Using AgentAssist {#using-agentassist}
+### Familiarity with Keyboard Navigation
+AgentAssist is designed to enhance speed and efficiency, with a strong focus on **keyboard-based navigation**. While the application includes a Graphical User Interface (GUI), its full potential is unlocked when you use **keyboard commands**. Therefore, it is important to:
 
-AgentAssist's command box is your portal to managing contacts efficiently. Here’s how you can use it to streamline your tasks:
+- Be comfortable navigating applications using the keyboard.
```

Comment:
Perhaps this line is not needed.

Reply:
Good catch!

Reply:
Agreed, the title of that section implies it anyways.